dtfs-central-api/src/v1/controllers/tfm/deal/tfm-get-deals.controller.js

```diff
@@ -2,11 +2,17 @@ const moment = require('moment');
 const db = require('../../../../drivers/db-client');
 const CONSTANTS = require('../../../../constants');
 const getObjectPropertyValueFromStringPath = require('../../../../utils/getObjectPropertyValueFromStringPath');
+const getCrossDataModelProperty = require('../../../../utils/getCrossDataModelProperty');
+
+const isEmpty = (value) => {
+  if (value === null) return '';
+  return value;
+};
 
 const sortDeals = (deals, sortBy) =>
   deals.sort((xDeal, yDeal) => {
-    const xField = getObjectPropertyValueFromStringPath(xDeal, sortBy.field);
-    const yField = getObjectPropertyValueFromStringPath(yDeal, sortBy.field);
+    const xField = isEmpty(getObjectPropertyValueFromStringPath(xDeal, getCrossDataModelProperty(xDeal, sortBy.field)));
```

Comment:
Why do we need `isEmpty`? It's a bit hard to read this line

Reply:
To find values which are null, if they are then give a unicode value which improves the sorting. I can break into multiple lines no problem.